examples/api-express/src/index.ts

```diff
@@ -7,13 +7,24 @@ import { createApp } from '@davinci/core';
 import { ExpressHttpServer } from '@davinci/http-server-express';
 import { HealthChecksModule } from '@davinci/health-checks';
 import { OpenAPIModule } from '@davinci/openapi';
+import addErrors from 'ajv-errors';
+import { createAjvValidator } from '@davinci/http-server';
 import { CustomerController } from './api/customer';
 
 const app = createApp();
 const contextFactory = ({ request }) => ({ accountId: request.headers['x-accountid'] });
 
 app.registerController(CustomerController).registerModule(
-	new ExpressHttpServer({ contextFactory }),
+	new ExpressHttpServer({
+		contextFactory,
+		validationFactory: createAjvValidator({
+			ajvOptions: {
+				header: { coerceTypes: true },
+				query: { coerceTypes: true }
+			},
+			plugins: [[addErrors as any]]
```

Comment:
Why do we need to cast to any? I imagine plugins would have a common interface or base type

Reply:
For some reason TS is not getting the types correctly. I asked @pierissimo about this and he will take a look soon

Reply:
probably a mismatch between the Ajv version installed as dependency of the `api-express` example, and the Ajv version installed as devDependency in `http-server`

Reply:
The version difference is just a patch and by the changelog I don't see anything relevant.
I think it could be a problem of the ajv-error plugin, since I tried with the ajv-formats plugin and it works in this case

Reply:
You're right @pierissimo. Bumping ajv in the examples from 8.11.0 to 8.11.2 fixed the problem